open-ended-capstone/second_etl.py

```diff
@@ -0,0 +1,194 @@
+import random
```

Comment:
code is not readable—variables, classes and functions are all mixed up.

no clearly visible entry point